### PR TITLE
Salmon install and commands

### PR DESCRIPTION
**Pull request by johnsolk**
Install and commands work. Still working on extracting counts from quant.sf files.

### CODE REVIEW

File: 6-expression.rst [hunk 1]
```diff
@@ -1,4 +1,108 @@
+==========================================
 6. Analyzing RNAseq expression with Salmon
 ==========================================
 
-Salmon!
+We will use `salmon <http://salmon.readthedocs.org/en/latest/>`__ to
+quantify differential expression. `Salmon
```
Comment (standage) [inline 6-expression.rst]: I think "quantify expression" would be more accurate than "quantify differential expression", no?

File: 6-expression.rst [hunk 2]
```diff
@@ -1,4 +1,108 @@
...
+However, they're not in a convenient location / format for edgeR to use;
+let's fix that.
+
+Now, grab the script...
+
+::
+   
+   curl -L -O https://raw.githubusercontent.com/dib-lab/eel-pond/DE/gather-counts.py
+
+(@CTB let's copy that script into this repo, @LJC Done.)
```
Comment (standage) [inline 6-expression.rst]: Remove this comment?

**Review by standage — COMMENTED** [review]
LGTM, aside from the two comments.

File: 6-expression.rst (hunk 1 above)
Comment (johnsolk) [reply 6-expression.rst]: Right, thank you. Fixed.

File: 6-expression.rst (hunk 2 above)
Comment (johnsolk) [reply 6-expression.rst]: Removed

**Review by standage — APPROVED** [review]

**Pull request merged**